board/views.py

```diff
@@ -24,18 +28,79 @@ def __init__(self):
         self.platforms = res_platforms
 
 
+class BaseGameView:
+    def __init__(self, user):
+        self.user = user
+
+    def _get_context(self, game_id, is_favourite):
+        if Game.is_exist(game_id):
+            game = Game(game_id)
+            tweets = Tweet.get_tweets(game_id)
+        else:
+            game = GameAPI(game_id)
+            tweet_ids = twitter_wrapper.get_tweets_by_string(game.slug)
+            if tweet_ids:
+                tweets = [TweetAPI(tweet_id) for tweet_id in tweet_ids]
+            else:
+                tweets = None
```

Comment:
If `tweet_ids` is empty, then tweets will be an empty list.
```suggestion
            tweets = [TweetAPI(tweet_id) for tweet_id in tweet_ids]
```